test/integration/features/cmd-config.feature

```diff
@@ -0,0 +1,242 @@
+    | iso-url         | file://home/me/Downloads/my_handmade_b2d.iso                                                    | file://home/me/Downloads/my_handmade_b2d.iso                                                    |
+    | iso-url         | b2d                                                                                             | b2d                                                                                             |
+    | iso-url         | centos                                                                                          | centos                                                                                          |
+
+  Scenario: Unsetting non-existing key
+     When executing "minishift config unset i-do-not-exist" succeeds
+     Then exitcode should equal "0"
+
+  Scenario: Getting non-existing key
+     When executing "minishift config get does-not-exist"
+     Then stdout should contain "<nil>"
+
+  Scenario Outline: Setting values, getting values and keeping them
+  Setting values, not unsetting them so they will be used on next Minishift start.
+  Not every key possible is being tested only those which are less complicated,
+  for example the http-proxy key is being tested in separate feature file.
+     When executing "minishift config set <property> "<value>"" succeeds
+     Then stdout of command "minishift config get <property>" is equal to "<expected>"
+
+  Examples: Values to be used on next Minishift start
+    | property          | value              | expected             |
+    | memory            | 3500               | 3500                 |
+    | disk-size         | 25g                | 25g                  |
+    | cpus              | 3                  | 3                    |
+    | docker-env        | FOO=BAR,hello=hi   | [FOO=BAR hello=hi]   |
+    | docker-opt        | dns=8.8.8.8        | [dns=8.8.8.8]        |
+    | insecure-registry | test-registry:5000 | [test-registry:5000] |
+
+  Scenario: Minishift informs about starting with correct setup of memory, disk and CPU
+  Note: Minishift rounds the values for the report to make it more readable.
+        However original non-rounded values are used for the startup.
+     When executing "minishift start" succeeds
+     Then stdout should match "Memory\s*:\s*3 GB"
+     Then stdout should match "Disk size\s*:\s*25 GB"
+     Then stdout should match "vCPUs\s*:\s*3"
+      And Minishift should have state "Running"
+
+  Scenario: Checking that memory value was applied
+     When executing "minishift ssh -- less /proc/meminfo" succeeds
+     Then stdout should match "MemTotal:\s*35[0-1][0-9]{4} kB"
+
+  Scenario: Checking that disk-size value was applied
+     When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
+     Then stdout should match "Disk \/dev\/sda: 2[4-6]\.?[0-9]{0,2} GB"
+
+  Scenario: Checking that cpus value was applied
+     When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
```

Comment:
how about `nproc` or `lscpu` ?
If these are possible on B2D it simplifies the test